src/PerformanceTests/Utils/Variables/Persistence.cs

```diff
@@ -10,6 +10,8 @@ public enum Persistence
         RavenDB_Embedded,
         //MSMQ,
         Azure,
-        Sql
+        Sql,
+        Sql_Azure,
```

Comment:
Practically speaking this could have been another PR right?

Reply:
Strictly taken all 3 commits could have been 3 separate PR's.